### DISCUSSION TITLE
we are improving py-contributors official website

### DISCUSSION THREAD

**Discussion by codeperfectplus** [Py-Contributors/support#23] [Project Discussion]
**There are a few upcoming changes on the site. Contributions are welcome.**

1. In-home Page, add more information about the organization, Like Organization AIM, Objective, founded year, Joining Link
2. Improve the Projects page card design. it's very dull right now. the color combination is not that good.
3. Create a button for Light Dark mode. There should be a button in the right corner to switch the theme.
4. Rewriting the join us page. It's currently from an open-source perspective. we have to make some changes from the py-contributors perspective.
5. To make site more responsive (suggested by @bandya2003 )

**Suggestions are welcome.**

**Comment by codeperfectplus** [comment]
@reveurguy 
@bandya2003

**Reply by bandya2003** [reply]
We can update the footer linking the Github repo and discord link for the community discussions !

**Reply by codeperfectplus** [reply]
![image](https://user-images.githubusercontent.com/54245038/198589055-16412e83-d466-4df4-bcf7-ac3371cfcae9.png)
 we can add like this

**Reply by bandya2003** [reply]
If possible please can we change the background image of the Dropdown Menu the Bubbles kind of look distracting.

**Reply by bandya2003** [reply]
> ![image](https://user-images.githubusercontent.com/54245038/198589055-16412e83-d466-4df4-bcf7-ac3371cfcae9.png) we can add like this

Yeah like this 👍

**Reply by codeperfectplus** [reply]
yeah, we can experiment with the background image.

**Reply by codeperfectplus** [reply]
@bandya2003 you can open an issue for what changes you are going to make and start working on the changes.
